src/Actions/Products/ImportProducts.php

```diff
@@ -90,8 +92,6 @@ public function delete(?string $store = null): void
             $productSkus = collect();
 
             $productQuery = $productModel::selectOnlyIndexable()
```

Comment:
We only need the sku and name right? Just select them explicitly instead of all indexable attributes
```suggestion
            $productQuery = $productModel::selectAttributes(['sku', 'name'])
```
Probably all the global scopes are also not needed which speeds up the query